desktop/main.js

```diff
@@ -147,6 +150,27 @@ const mainWindow = (() => {
                 titleBarStyle: 'hidden',
             });
 
+            if (!isDev) {
+                const newDotURL = isProduction ? 'https://new.expensify.com' : 'https://staging.new.expensify.com';
```

Comment:
It feels like it would be cleaner to have a single variable called `env` that would be either dev, staging, or production. Is there anyway we can use the ENVIRONMENT variable from the .env files?

Reply:
I agree one env variable would be cleaner.

Reply:
> It feels like it would be cleaner to have a single variable called env that would be either dev, staging, or production.

Let me look into this.

> Is there anyway we can use the ENVIRONMENT variable from the .env files?

_Maybe_, but the `.env` files are consumed using `react-native-config` and/or `react-web-config`. This is the main process, running in an Electron context, more akin to a Node.js environment than a web environment. It might be possible to use `dotenv` separately to consume the `.env` files, but we'd still need a way to tell Electron which environment it's running in.

Reply:
> Is there anyway we can use the ENVIRONMENT variable from the .env files?

Expanding on this some more, we'd have no way of knowing which `.env` file to use. For the main web app, we have a separate webpack config that specifies this for us. However, the code in `desktop/main.js` isn't touched by webpack. Still thinking of how this can be cleaned up.